### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## Math magicians: Tests
This is the Fifth project of the Math Magicians application. We Add tests for the component and logic JS file. The requirements of this project are:

- Using linters to free of errors.
- Using Gitflow
- Documented in professional way
- Write unit tests for the files `operate.js` and `calculate.js` using Jest.
- Use Jest snapshots to test the components.

### CODE REVIEW

File: src/components/__test__/CalculatorPage.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import CalculatorPage from '../CalculatorPage';
+
+it('Render the Calculator page user interaction', () => {
+  const tree = renderer.create(<CalculatorPage />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (tresorsawasawa) [inline src/components/__test__/CalculatorPage.test.js]: - [ ] As required for the project, kindly ensure that there are the usage of React Testing Library to simulate user interaction.

File: src/logic/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  test('Test calculator and operator module n1: 23 % n2: 5', () => {
+    const cal = {
+      next: '5',
+      total: '23',
+      operation: '%',
+    };
+    expect(calculate(cal, '=').total).toBe('3');
+  });
+});
```
Comment (tresorsawasawa) [inline src/logic/calculate.test.js]: - [ ] As required for the project, all the functionalities & scenarios of the `operate.js` and` calculate.js` modules  are covered with tests using Jest. Kindly ensure that you also add tests on the `+/-`, ...

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@
 # 📗 Table of Contents
```
Comment (tresorsawasawa) [inline README.md:16]: - [ ] Kindly ensure that you also fill the `Acknowledgments` and `Show your support` sections from your your README with correct and meaningful content

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Iqbal-Elham and @fithamlak 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 
- Use of pair-programming-o :heavy_check_mark: 


## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
Hi Team 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job working on the suggested changes 😉
- Tests are passing ✔️
- No linter error 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**